engine/lite/runtime/src/main/scala/pl/touk/nussknacker/engine/lite/metrics/dropwizard/LiteMetricRegistryFactory.scala

```diff
@@ -60,7 +59,15 @@ object LiteEngineMetrics extends LazyLogging {
   }
 
   case class CommonMetricConfig(prefix: Option[String],
-                                host: Option[String],
+                                instanceId: Option[String],
                                 environment: String,
                                 additionalTags: Map[String, String] = Map.empty)
 }
+
+object LiteMetricRegistryFactory {
+
+  def usingHostnameAsDefaultInstanceId = new LiteMetricRegistryFactory(sys.env.getOrElse(
+    "HOSTNAME",
+    throw new IllegalStateException("HOSTNAME environment variable unavailable")))
```

Comment:
Can we also have some method for adding a suffix (like port etc.), so we don't have to put this HOSTNAME logic twice?